### PR TITLE
Develop add doc all apis

### PR DESCRIPTION
**Pull request by VShveda**

### CODE REVIEW

File: book/serializers.py [hunk 1]
```diff
@@ -10,6 +11,7 @@ class Meta:
         fields = ["id", "title", "author", "cover", "inventory", "daily_fee"]
 
     def create(self, validated_data):
+        """Create and return a new book."""
```
Comment (SuskyiVolodymyr) [inline book/serializers.py:13]: maybe better "Create a book, if one does not exist yet, or combine"

File: book/serializers.py [hunk 2]
```diff
@@ -23,6 +25,7 @@ def create(self, validated_data):
         return Book.objects.create(**validated_data)
 
     def update(self, instance, validated_data):
+        """Update and return an existing book."""
```
Comment (SuskyiVolodymyr) [inline book/serializers.py:27]: maybe better "Updatea book, if one does not exist yet, or combine"

File: book/views.py [hunk 3]
```diff
@@ -9,3 +10,10 @@ class BookViewSet(viewsets.ModelViewSet):
     queryset = Book.objects.all()
     serializer_class = BookSerializer
     permission_classes = [IsAdminOrReadOnly]
+
+    def list(self, request, *args, **kwargs):
+        """List all books. If the user is an admin, all books are listed.
+        If the user is staff, all books are listed.
+        If the user is not admin or staff, only books belonging to the user are listed.
+        """
+        return super().list(request, *args, **kwargs)
```
Comment (SuskyiVolodymyr) [inline book/views.py:18]: i think it is not necessary, you can just remove these lines

File: borrowing/telegram_helper.py [hunk 4]
```diff
@@ -10,6 +10,11 @@
 
 
 def send_message(text):
+    """
+    Send a message to the Telegram chat specified by CHAT_ID.
+    Args: text (str): The message text to be sent.
+    Returns: dict: A dictionary with the result of the message sending.
```
Comment (SuskyiVolodymyr) [inline borrowing/telegram_helper.py:16]: too many information, "Send a message to the Telegram chat" will be enough

**Review by SuskyiVolodymyr — CHANGES_REQUESTED** [review]

File: borrowing/telegram_helper.py [hunk 5]
```diff
@@ -11,9 +11,7 @@
 
 def send_message(text):
     """
-    Send a message to the Telegram chat specified by CHAT_ID.
-    Args: text (str): The message text to be sent.
-    Returns: dict: A dictionary with the result of the message sending.
+    Send a message to the Telegram cha`t will be enough.
```
Comment (SuskyiVolodymyr) [inline borrowing/telegram_helper.py:14]: 😂

**Review by SuskyiVolodymyr — CHANGES_REQUESTED** [review]

**Review by SuskyiVolodymyr — APPROVED** [review]

**Review by AntonKorinchuk — APPROVED** [review]
Good job

**Pull request merged**